README.md

```diff
@@ -35,6 +35,19 @@ services:
       - DISCORD_USER_ID= # Discord user ID of the user you want Aoede to follow
 ```
 
+**Docker**:
+```env
+# .env
+ENV DISCORD_TOKEN=
```

Comment:
here should be no `ENV` just `DISCORD_TOKEN=XY`, the same for the lines below

Reply:
My bad...

Reply:
Fixed in 929c989798a09cd47f5a690944d22443ecf92f44